enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/CausalClusteringSettings.java

```diff
@@ -137,7 +138,8 @@
     public static final Setting<Integer> minimum_core_cluster_size_at_runtime =
             buildSetting( "causal_clustering.minimum_core_cluster_size_at_runtime", INTEGER, "3" ).constraint( min( 2 ) ).build();
 
-    @Description( "Network interface and port for the transaction shipping server to listen on." )
+    @Description( "Network interface and port for the transaction shipping server to listen on. Please note that it is also possible to run the backup " +
+            "client against this port so always limit access to it via the firewall and configure an ssl policy. Please use dbms.backup.address for backups." )
```

Comment:
Perhaps remove the "Please use dbms.backup.address for backups" recommendation. I think you could state something similar in regular documentation, but perhaps slightly different. Something regarding that the additional backup port is possible to use for backwards compatibility and operational flexibility, but it is not a necessity.